source/server-selection/server-selection-tests.rst

```diff
@@ -282,6 +282,8 @@ Prose Test
 Multi-threaded and async drivers MUST also implement the following prose test:
 
 1. Configure a sharded cluster with two mongoses. Use a 4.2.9 or newer server version.
+   This test should not be run using TLS or authentication to reduce the chance of
+   a false test failure.
```

Comment:
Why does TLS/authentication affect the probability of a failure? It'd be helpful to mention the explanation in the test.

Reply:
I'm not actually certain--I think maybe they contributed to the random latency spikes that caused one server to fall out of the latency window? Now that we have a high localThresholdMS that might not be a problem anymore. For reference, I just tried out the new version of the test with auth + TLS enabled and had no failures, so maybe this isn't required anymore. I know .Net said skipping auth + TLS helped a lot, so I wonder if @BorisDog could try this new version out with auth/TLS enabled and see if skipping is still necessary.

Reply:
Yes, will do.

Reply:
With the proposed changes, the test passes successfully on auth topologies. I believe that non-auth requirement can be removed.

Reply:
Ok nice, updated to remove the requirement.

Reply:
Thank you, Boris, for checking!